openetl_utils/pipeline_utils.py

```
@@ -303,3 +309,43 @@ def coerce_inferable_columns(df: pd.DataFrame, spark_session: SparkSession, logg
             logger.warning(f"Unable to infer type for column {col}: {e}")
             df[col] = df[col].astype(str)
     return df
+
+
+def map_spark_type_to_sqlalchemy(spark_type, max_len=None):
+    from sqlalchemy import Integer, BigInteger, Float, Boolean, DateTime
```

Ruff (0.11.9)

Redefinition of unused `Integer` from line 25

Remove definition: `Integer`

(F811)